flowauth/frontend/cypress/fixtures/openapi.json

```diff
@@ -0,0 +1,3173 @@
+{
+  "components": {
```

Comment:
Should this file be here? It seems to be a broken spec, and better to use the api_sec.json which symlinks to the one in the integration tests that gets checked against the one generated by flowapi.